app/elements/submit-page.html

```diff
@@ -815,47 +768,51 @@ <h4 class="role__name"><img src="../images/icons/share.svg" alt="Share" class="r
       <button on-tap="changeSection" class="section__back" data-display="main" data-hide="startup">Back</button>
       <div class="mdl-layout-spacer"></div>
       <div class="container mdl-cell mdl-cell--12-col">
-        <echo-html html$="{{startups.submit_startup_2_1}}"></echo-html>
+            <h3 class="section__subheader section__subheader-startup">
+                <echo-html html$="{{submit.startups.title}}"></echo-html>
+            </h3>
+            <p class="section__description section__description-startup">
+                <echo-html html$="{{submit.startups.description}}"></echo-html>
+            </p>
+            <img src="../images/icons/startups_steps.svg" alt="Steps for startup" class="section__icon--wide">
+            <h3 class="section__subheader section__subheader-startup">
+                <echo-html html$="{{submit.startups.process_title}}"></echo-html>
+            </h3>
+            <p class="section__description section__description-startup">
+                <echo-html html$="{{submit.startups.process_description}}"></echo-html>
```

Comment:
instead of pulling whole content collection, I'd suggest to have 2 or 3 elements now:
submitStartups, submitContryOffices, submitCommon (or sth similar) - what do you think?